src/Microsoft.HttpRepl/Resources/Strings.resx

```diff
@@ -350,4 +350,8 @@ When +history option is specified, commands specified in the text file will be a
   <data name="SetBaseCommand_MustSpecifyServerError" xml:space="preserve">
     <value>Must specify a server</value>
   </data>
+  <data name="RealFileSystem_Error_InvalidExtension" xml:space="preserve">
+    <value>If specified, {0} must begin with a period.</value>
```

Comment:
Is this an actionable user message?  This seems more like a developer-oriented message.

Reply:
You're correct. Your point being, I assume, that it doesn't need to be in `Strings`.